core/sr-primitives/src/traits.rs

```diff
@@ -475,9 +475,10 @@ pub trait Hash: 'static + MaybeSerializeDebug + Clone + Eq + PartialEq {
 #[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
 pub struct BlakeTwo256;
 
-impl Hash for BlakeTwo256 {
+impl self::Hash for BlakeTwo256 {
```

Comment:
Why we need this self here?

Reply:
Clippy thinks that `Hash` alone is the std `Hash`, which triggers https://rust-lang.github.io/rust-clippy/master/index.html#derive_hash_xor_eq

Reply:
Same.

Reply:
Please revert this one.